disneyApi/src/hooks/useEscKey.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react';
+const useEscKey = (setState) => {
+	useEffect(() => {
+		const closeModalByEsc = (e) => {
```

Comment:
The function name closeModalByEsc could be more descriptive. Consider renaming it to handleEscKeyPress for better clarity.